hbase-balancer/src/main/java/org/apache/hadoop/hbase/master/balancer/BalancerClusterState.java

```diff
@@ -290,7 +294,9 @@ public String getRack(ServerName server) {
     }
 
     numTables = tables.size();
+    LOG.debug("number of tables = {}", numTables);
```

Comment:
nit: for next time, we capitalize log messages and if you look at other logs, there is no space around '=' when used in log messages.